bin/create-dmg

```diff
@@ -1,46 +0,0 @@
-#!/bin/bash
```

Comment:
why are we deleting upstream files?

Reply:
The reaction to delete this file is "natural". But consider this:
imagine we forked the upstream linux source code to build our own appliance.
We would not find and then delete all the kernel source files for unsupported platforms. We would just leave them there :-)

Although "upstream" is currently inactive, the moment we merge either from the current upstream or from another fork, we risk having conflicts (this particular one will not cause conflicts, but still).

The rule of thumb is:
If the fork is soft (which is the case, for the time being), change only what you need.
If the fork is hard, change what you want.

Reply:
OK, then.

Reply:
The file has been restored. There are no linting errors when running with their version of go and golangci-lint.